repos/system_upgrade/common/models/rpmtransactiontasks.py

```diff
@@ -10,6 +10,8 @@ class RpmTransactionTasks(Model):
     to_keep = fields.List(fields.String(), default=[])
     to_remove = fields.List(fields.String(), default=[])
     to_upgrade = fields.List(fields.String(), default=[])
+    modules_to_enable = fields.List(fields.Model(Module), default=[])
+    modules_to_reset = fields.List(fields.Model(Module), default=[])
```

Comment:
we should finally cover model's fields by docstrings. but not blocker for this pr.